<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FestiveViewControls</class>
 <widget class="QWidget" name="FestiveViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Festive Plugin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_1">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step1: Load DICOM</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_2">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step2: Process Images</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button_2_1">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Convert to Nifti</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_3">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step3: Create Mask</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button_3_1">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Save Mask</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_4">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step4: Establish Connection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_5">
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step5: Copy to Server</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_6">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step6: Run Reconstruction</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_7">
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step7: Download Result</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_r">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
